lobby-db/launch_db

```diff
@@ -3,12 +3,40 @@
 # Convenience script to get a local DB up and running on docker
 # with some sample data.
 
-set -eu
+set -u
+set -e
 
-path=$(dirname $0)
+path=$(dirname "$0")
+"$path/run_docker"
 
-$path/run_docker
-sleep 5
-$path/run_flyway
-$path/load_sample_data
+set +e
+echo -n "Waiting for Database start"
+status=1
+tryAttempt=0
+while [ "$status" != 0 ]; do
+  echo 'select 1' \
+     | psql -h localhost -U lobby_user lobby_db 2> /dev/null \
+     | grep -q '1 row'
+  status=$?
```

Comment:
In bash you can inline this check into the while condition.
This way there's no need to declare a status variable at all.

See https://stackoverflow.com/a/5274386

Reply:
Indeed, it's preferred even. Something was going wrong with it though... I gave up after 20 minutes.

Reply:
Just needed to spend a moment longer, managed to get it correctly formatted/syntaxed : )

It actually really helped to have had a working version first and then to iterate to get it right, my first attempt was to have the 'while' loop directly invoke the corresponding command via a variable.. 

Updated 223d6eb